app/Http/Controllers/GenreController.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Genre;
+use Exception;
+use Illuminate\Http\Request;
+
+class GenreController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        try {
+            $genres = Genre::paginate(10);
+            return response()->json($genres, 200);
+        }catch (Exception $e){
+            return response()->json($e->getMessage(),400);
```

Comment:
@damir-tahirovic As we are developing only api maybe it would be better to return some user-friendly message.